### DISCUSSION TITLE
article/moyu

### DISCUSSION THREAD

**Discussion by giscus** [JobsQzhang2018/Discussions-Backup#12] [Announcements]
# article/moyu

工作烦躁？想偷懒？还不被领导发现？不妨来看下这款100%好评小软件……

https://blog.zzc.ac.cn/article/moyu

**Comment by JobsQzhang2018** [comment]
欢迎评论~~~

**Comment by tangly1024** [comment]
真是令人大开眼见的软件~ 可惜没有mac版本😢

**Reply by JobsQzhang2018** [reply]
确实~   回头找找看，要不要考虑虚拟机或者双系统呢？  有时候为了摸鱼，真的可以不择手段
但是，为了自己，真没必要摸鱼~~我是单纯的觉得这个东西做得好玩，分享下~
